compiler/include/symbol.h

```diff
@@ -91,6 +91,20 @@ typedef std::bitset<NUM_FLAGS> FlagSet;
 ArgSymbol* tiMarkForIntent(IntentTag intent);
 ArgSymbol* tiMarkForTFIntent(int tfIntent);
 
+//
+// ForallIntentTag: a task- or forall-intent tag
+//
+enum ForallIntentTag {
+  TFI_DEFAULT, // aka TFI_BLANK
+  TFI_CONST,
+  TFI_IN,
+  TFI_CONST_IN,
+  TFI_REF,
+  TFI_CONST_REF,
+  TFI_REDUCE,
+};
+
+const char* tfiTagDescrString(ForallIntentTag tfiTag);
```

Comment:
getForallIntentTagDescription ?

Reply:
I don't like "get". Also, we have intentDescrString() and retTagDescrString(). So how about forallIntentDescrString() ? I also like forallIntentTagDescription(). Which would you prefer?

Reply:
forallIntentDescrString() is reasonable (since other code looks like that) but I'd slightly prefer forallIntentTagDescription().